msteams-platform/toolkit/add-API-connection.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Connect to existing APIs
+author: MuyangAmigo
+description:  Describes connection to existing APIs
+ms.author: zhany
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/15/2022
+---
+
+# Connect to existing APIs
+
+Teams Toolkit helps you to access existing APIs for building Teams applications. These APIs are developed by your organization or third-party.
+
+## Advantage
+
+Teams Toolkit helps you bootstrap sample code to access the APIs if you don't have language appropriate SDKs to access these APIs.
+
+## Connect to the API
+
+When you use Teams Toolkit to connect to an existing API, Teams Toolkit performs the following function:
+
+* Generate sample code under `./bot` or `./api` folder
+* Add a reference to the `@microsoft/teamsfx` package to `package.json`
+* Add application settings for your API in  `.env.teamsfx.local` that configures local debugging
+
+### Connect to API in Visual Studio Code
+
+* You can add API connection using Teams Toolkit in Visual Studio Code:
+
+    1. Open Microsoft Visual Studio Code.
+    2. Select Teams Toolkit :::image type="icon" source="../assets/images/teams-toolkit-v2/add-API/api-add-icon.png" alt-text="api icon"::: from the left navigation bar.
+    3. Select **Add features** under **DEVELOPMENT**:
+
+        :::image type="content" source="../assets/images/teams-toolkit-v2/add-API/api-add-features.png" alt-text="api add features":::
+
+       * You can also open the command palette and enter **Teams: Add cloud resources**.
+
+    4. From the pop-up, select the **API Connection** you want to add to your Teams app project:
+
+        :::image type="content" source="../assets/images/teams-toolkit-v2/add-API/api-select-features.png" alt-text="api select features":::
```

Comment:
the API selected screenshot is selecting single sign on. please change the screenshot. Thank you~

Reply:
Updated as mentioned.